interfaces/cython/cantera/units.pyx

```diff
@@ -18,8 +19,40 @@ cdef class Units:
         if name:
             self.units = CxxUnits(stringify(name), True)
 
-    def __repr__(self):
-        return f"<Units({pystr(self.units.str())}) at {id(self):0x}>"
+    def __repr__(self) -> str:
+        return f"<Units({str(self)}) at {id(self):0x}>"
+
+    def __str__(self) -> str:
+        return pystr(self.units.str())
+
+    def dimension(self, primary: str) -> float:
+        """The dimension of the given unit component.
+
+        .. versionadded:: 3.0
+
+        :param primary:
+            a string with the desired unit component. One of ``"mass"``,
```

Comment:
While I believe that this is resolved, I noticed that many comparable docstring entries start with uppercase?